content/docs/user-guide/project-structure/configuration.md

```diff
@@ -111,6 +111,22 @@ within:
   <abbr>DVC files</abbr> created or modified by DVC commands. The files will not
   be committed. Accepts values `true` and `false` (default).
 
+- `core.site_cache_dir` - specify a custom location for misc temporary files.
```

Comment:
let's put a link to the `Site cache dir` section below? (like Read more here ...)

Then we might even move the admon below to that section and simplify it here, wdyt?

Reply:
Done